tests/unit/pkcs7.js

```diff
@@ -518,6 +519,23 @@ var support = require('./support');
       ASSERT.equal(p7.content, 'Just a little test');
     });
 
+    it('should export message to PEM with content in DER-format', function() {
+      var p7 = PKCS7.createEnvelopedData();
+      p7.addRecipient(PKI.certificateFromPem(_pem.certificate));
+      p7.content = UTIL.createBuffer(ASN1.toDer(ASN1.create(ASN1.Class.UNIVERSAL, ASN1.Type.OCTETSTRING, false, 'Just a little test')));
+      p7.encrypt();
+
+      var pem = PKCS7.messageToPem(p7);
+
+      // convert back from PEM to new PKCS#7 object, decrypt, and test
+      p7 = PKCS7.messageFromPem(pem);
+      p7.decrypt(p7.recipients[0], PKI.privateKeyFromPem(_pem.privateKey));
+      var asn1 = ASN1.fromDer(p7.content);
+      ASSERT.equal(asn1.value, 'Just a little test');
```

Comment:
This test case looks good from what I can tell. Will look more closely when I get a chance.